SmartDeviceLink-iOS/SmartDeviceLinkTests/RPCSpecs/FactorySpecs/SDLRPCRequestFactorySpec.m

```diff
@@ -847,6 +847,68 @@
     });
 });
 
+describe(@"SDLAlertManeuver Tests", ^ {
+    it(@"Should build correctly", ^ {
+
+        NSArray* softButtons = @[[[SDLSoftButton alloc] init]];
+        NSArray* ttsChunks = @[[[SDLTTSChunk alloc] init]];
+        SDLAlertManeuver* message = [SDLRPCRequestFactory buildAlertManeuverwithTTSchunks:ttsChunks softButtons:softButtons correlationID:@1234];
+
+        expect(message.ttsChunks).to(equal(ttsChunks));
+        expect(message.softButtons).to(equal(softButtons));
+        expect(message.correlationID).to(equal(@1234));
+
+        message = [SDLRPCRequestFactory buildAlertManeuverwithTTSchunks:ttsChunks softButtons:softButtons correlationID:@99999];
+        expect(message.ttsChunks).to(equal(ttsChunks));
+        expect(message.softButtons).to(beNil());
+        expect(message.correlationID).to(equal(@99999));
```

Comment:
It's fine for now because many other tests are like this, but for your own knowledge, this is an incorrect formatting for the tests. The tests should be formatted so that each `it` block only contains a single `expect` assertion. See this PR for an example of proper formatting: https://github.com/smartdevicelink/sdl_ios/pull/273